### ISSUE TITLE
Rubric questions: allow weight for each cell: Migrate legacy data into new format

### ISSUE THREAD

**Issue by sukanta-27** [TEAMMATES/teammates#9007]
**Context:** [Rubric questions: allow weight for each cell](https://github.com/TEAMMATES/teammates/issues/7224)
 
Migrate legacy data for rubric weights into new format.

Should be done after #8961 is merged.

**Note:** The next step of data migration is to remove the code that deals with legacy data. For this step see #9026

**Comment by sukanta-27** [comment]
@xpdavid Should we create a new issue for this bug? I deleted the old branch.

**Comment by xpdavid** [comment]
Open a new branch based on the master.
